Kitodo/src/main/java/org/kitodo/production/services/data/ProcessService.java

```diff
@@ -1157,13 +1157,13 @@ public URI getProcessURI(Process process) {
      * Returns a URI that identifies the process. The URI has the form
      * {@code mysql://?process.id=42}, where {@code 42} is the process ID.
      *
-     * @param id
+     * @param aLongerVariableNameForTheIdToSatisfyCodacy
      *            process ID for which a URI is to be formed that identifies it
      * @return a URI that identifies the process
      */
-    public URI getProcessURI(Integer id) {
+    public URI getProcessURI(Integer aLongerVariableNameForTheIdToSatisfyCodacy) {
```

Comment:
Really? What is wrong with `processId`?

Reply:
Or simply `identifier`?

Reply:
Yes, `id` is too short and was rejected.

Reply:
I have renamed it now in `processId` and hope that works. But if we no longer want to use `id` in the code, we should also rename all the corresponding fields, database rows, and methods, such as `getById()`. But maybe this rule is too strict. The use of `id` is widespread in Java.